content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -86,10 +86,16 @@ def vm_prov_option_value(prov_option, options_array = [])
                :flavor        => flavor_obj(@miq_request.get_option(:instance_type)),
                :number_of_vms => get_option_value(@miq_request, :number_of_vms),
                :cloud         => vm_provision_cloud?}
+  # number_of_vms doesn't exist for VmReconfigureRequest
+  args_hash[:number_of_vms] = 1 if @reconfigure_request
 
   case prov_option
   when :vm_memory
-    requested_memory(args_hash, @miq_request.source.vendor)
+    if @reconfigure_request
```

Comment:
@billfitzgerald0120 @tinaafitz 
Can we add a vendor method
```
def vendor
  @reconfigure_request ? @vm.vendor : @miq_request.source.vendor
end
```

The this becomes
```
   when :vm_memory
        requested_memory(args_hash, vendor)
```